pkg/operator/staticpod/controller/revision/deployment_controller.go

```diff
@@ -180,6 +184,12 @@ func (c RevisionController) createNewRevision(revision int32) error {
 		UID:        statusConfigMap.UID,
 	}}
 
+	for _, name := range c.optionalConfigMaps {
+		_, _, err := resourceapply.SyncConfigMap(c.kubeClient.CoreV1(), c.eventRecorder, c.targetNamespace, name, c.targetNamespace, nameFor(name, revision), ownerRefs)
+		if err != nil {
+			return err
```

Comment:
uuh, how is optional?

Reply:
does not return err if obj == nil

Reply:
Ah sorry I was thinking this was `ApplyConfigMap`.